docs/context-aware-config.md

```diff
@@ -13,92 +13,90 @@ Context Aware Config (abbreviated as CAC) is the foundational service of the Sup
 
 ## Concepts
 ---
+
+Context-Aware-Config is made of 4 key abstractions default-configs, dimensions, context and overrides.
+
+
 ### Default Configs
 
-Default Configs are a fundamental concept to CAC. Default Configs are key value pairs that help CAC model the entity/configuration it is trying to manage and serve. Think of default configs as the best assumption we can make about our configs.
+Default Configs is the set of all possible configuration keys of your application along with their default values.  One may think of default configs as the best assumption we can make about our configs.
 
-Example:
-Using our car classification example, we can establish defaults like:
-1. Number of doors - 4 (most cars do have 4 doors)
-2. Engine - Internal Combustion
-3. Chassis - Hatchback
-4. Color - Silver 
-5. interiors - leather
+Let us use a simple cab ride-hailing application that operates in different cities and capture some attributes that we might need configurable at a per city level.
```

Comment:
@knutties can we link to the example code so that this folks can run the example too while reading this? Should we attach images or video?

Reply:
I am thinking I should move the demo-app as a crate in here.  Or are you saying point it to mine for now and when it is moved - update the docs to point to local crate?